asciidoc/courses/llm-fundamentals/modules/1-introduction/lessons/3-grounding/questions/1-rag-operation.adoc

```diff
@@ -0,0 +1,24 @@
+[.question]
+== 1.  RAG's Operation
+
+Which of the following best describes how Retrieval Augmented Generation (RAG) operates?
+
+* [ ] It trains the LLM on new data every time there is an update.
+* [ ] It solely uses external data sources to generate responses.
+* [*] It enhances LLM responses by combining the model with external retrieval mechanisms.
```

Comment:
too vague, make it more concrete
it doesn't combine the model, it passes the right,relevant external information as part of the prompt to the model